src/test/java/emissary/util/FlexibleDateTimeParserTest.java

```diff
@@ -420,6 +420,7 @@ void parse_yyyy_DDD() {
     @Test
     void testCleanDateString() {
         test("2016-01-04 18:20<br>", EXPECTED_NO_SECS, "HTML");
+        test("2016-01-04 18:20<br>br>", EXPECTED_NO_SECS, "HTML");
```

Comment:
I added this test because the previous rule failed on this case but the update allows it to pass successfully. I don't know how often something like this would happen or if this illustrates the proper intent of the original regex, but this new behavior seems reasonable given what it's trying to clean.